src/test/java/nextstep/line/acceptance/LineAcceptanceTest.java

```diff
@@ -0,0 +1,325 @@
+    }
+
+    /**
+     * Given: 특정 지하철 노선이 등록되어 있고,
+     * When: 관리자가 해당 노선을 삭제하면,
+     * Then: 해당 노선이 삭제되고 노선 목록에서 제외된다.
+     */
+    @DisplayName("노선 삭제 요청은, 노선 삭제를 요청하면 전체 노선을 조회했을 때 해당 노선이 제외된다.")
+    @Test
+    void deleteLineTest() {
+        // given
+        ExtractableResponse<Response> createdLineResponse = createLine(신분당선_PARAM);
+
+        // when
+        ExtractableResponse<Response> response = deleteLine(getId(createdLineResponse));
+
+        // then
+        assertResponseCode(response, HttpStatus.NO_CONTENT);
+        assertThat(getNames(findLines())).doesNotContain(신분당선);
+    }
+
+    /**
+     * Given: 특정 구간이 등록되어 있고,
+     * When: 구간을 추가하면,
+     * Then: 노선 조회시 구간이 등록되어있다.
+     */
+    @DisplayName("노선 구간 추가 요청은, 특정 노선에 구간 추가 요청을 하면 해당 노선 조회시 등록된 구간이 조회된다.")
+    @Test
+    void addSectionTest() {
+        List<Arguments> fixtures = addSectionFixtures();
+
+        for (Arguments arguments : fixtures){
+            // given
+            ExtractableResponse<Response> createdLineResponse = createLine((Map<String, Object>) arguments.get()[0]);
+            Map<String, Object> sectionParams = (Map<String, Object>) arguments.get()[1];
+            List<Long> expectedStationIds = (List<Long>) arguments.get()[2];
+
+            // when
+            ExtractableResponse<Response> response = addSection(getId(createdLineResponse), sectionParams);
+
+            // then
+            assertResponseCode(response, HttpStatus.CREATED);
+            List<Long> stationsIds = getStationIds(getId(createdLineResponse));
+            assertThat(stationsIds).isEqualTo(expectedStationIds);
+        }
+    }
+
+    private List<Arguments> addSectionFixtures() {
+        return List.of(
+            Arguments.of(신분당선_PARAM, 홍대역_강남역_구간_PARAM, List.of(분당역_ID, 홍대역_ID, 강남역_ID)),
+            Arguments.of(분당선_PARAM, 홍대역_분당역_구간_PARAM, List.of(홍대역_ID, 분당역_ID, 강남역_ID)),
+            Arguments.of(신분당선_PARAM, 분당역_성수역_구간_PARAM, List.of(분당역_ID, 성수역_ID, 홍대역_ID))
+        );
+    }
+
+    /**
+     * When: 존재하지 않는 노선에 구간을 등록하면,
+     * Then: 오류를 응답한다.
+     */
+    @DisplayName("존재하지 않는 노선에 구간 등록 요청을하면 오류가 발생한다.")
+    @Test
+    void notExistLineExceptionTest() {
+        // when
+        ExtractableResponse<Response> response = addSection(1L, 홍대역_강남역_구간_PARAM);
+
+        // then
+        assertResponseCode(response, HttpStatus.NOT_FOUND);
+    }
+
+    /**
+     * Given: 특정 지하철 노선이 등록되어 있고,
+     * When: 존재하지 않는 역이 포함된 구간을 등록하면,
+     * Then: 오류를 응답한다.
+     */
+    @DisplayName("존재하지 않는 역이 포함된 구간을 노선에 등록하면 오류가 발생한다.")
+    @Test
+    void notExistStationExceptionTest() {
+        // given
+        ExtractableResponse<Response> createdLineResponse = createLine(신분당선_PARAM);
+
+        // when
+        ExtractableResponse<Response> response = addSection(getId(createdLineResponse), 홍대역_서초역_구간_PARAM);
+
+        // then
+        assertResponseCode(response, HttpStatus.NOT_FOUND);
```

Comment:
```suggestion
        var createdLineResponse = createLine(신분당선_PARAM);

        // when
        var response = addSection(getId(createdLineResponse), 홍대역_서초역_구간_PARAM);

        // then
        assertResponseCode(response, HttpStatus.NOT_FOUND);
```

타입 추론을 사용하면 가독성이 더 좋아지겠죠? 타입 추론에 대해 요번 기회에 사용해 볼까요?

Reply:
확실히 코드가 더 간결해지고 가독성도 높아지겠네요! 이번에 다음 요청 때 사용해보겠습니다:)